exes/ethercat/inc/public/tfc/ec/devices/schneider/atv320/dbus-iface.hpp

```diff
@@ -314,6 +345,17 @@ struct controller {
         token);
   }
 
+  auto run_limitted(bool positive_speedratio) -> bool {
```

Comment:
```suggestion
  auto run_limited(bool positive_speedratio) -> bool {
```
could also be `run_from_limit` or `run_outoff_limit`